Chart.yaml

```
@@ -53,7 +53,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.0
+version: 0.2.1
```

we haven't been bumping version for this and this change doesn't affect output of the chart - just cleanup of the template.

```suggestion
version: 0.2.0
```

---

This change in fact fixes rendering issues.

To reproduce:

1. Navigate to the root of the project
1. Create a `values-web.yaml` with the following content:
```yaml
web:
  service:
    annotations: 
      service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
      service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy: 'ELBSecurityPolicy-TLS-1-2-2017-01'
```

2. Run the following command:
```bash
$ helm template temporal . -f values.yaml
```

3. The following error is displayed:
```
Error: YAML parse error on temporal/templates/web-service.yaml: error converting YAML to JSON: yaml: line 14: did not find expected key

Use --debug flag to render out invalid YAML
```

After the change in this PR, the template is generated successfully.

---

awesome thanks!